task3/index.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById("js-ul");
+
+for(let i = 1; i <= 2; i++){
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.textContent = "a"+i;
+    a.href = "a"+i+".html";
```

Comment:
私も先生から教わりましたが、[テンプレートリテラル](https://qiita.com/azukiazusa/items/b0d4a816142134ea7f34)を使って書いてみてください。
```suggestion
    a.textContent = `a${i}`;
    a.href = `a${i}.html`;
```

Reply:
とても勉強になりました！
次回から活用していきます。ありがとうございます。